src/writer/core_ui.py

```diff
@@ -44,6 +44,7 @@ class Component(BaseModel):
     parentId: Optional[str] = None
     handlers: Optional[Dict[str, str]] = None
     visible: Optional[Union[bool, str]] = None
+    visibleReversed: Optional[Union[bool]] = None
```

Comment:
It's just a bool, no ?
```suggestion
    visibleReversed: Optional[bool] = None
```